### ISSUE TITLE
[one-fits-all flavour] bad choice of colours

### ISSUE THREAD

**Issue by SteveALee** [cyclejs-community/create-cycle-app-flavors#9]
I'm not sure if you have control over the colours but dark blue on black is hard to read (also a WCAG accessibility failure on contrast ratio)

Effect these 2 lines
```
Project is running at http://localhost:8080/
404s will fallback to /index.html
```

Also the create-cycle app uses dark green on black which is better but hard to read

Perhaps a problem with the terminal emulator used in Windows 10 for cmd. I know colour handling is supposedly improved in the upcoming WIndows update.

**Comment by jvanbruegge** [comment]
the colors are the default webpack colors, it might be possible to change them

**Comment by jvanbruegge** [comment]
are the colors still bad?

**Comment by SteveALee** [comment]
Seems better with blue on black for home page - but I am in a dark coffee shop :)

**Comment by SteveALee** [comment]
At last! Microsoft have fixed. Updating insiders build now to test it :)

https://www.theverge.com/2017/8/2/16085806/microsoft-windows-console-color-scheme-update
